### PR TITLE
EEI story 2a

### PR DESCRIPTION
**Pull request by tim-lo**
Post:
image

Get:
image

### CODE REVIEW

**Comment by tim-lo** [comment]
To-Do: populate claimStatus field in db-writer

**Review by DanRoseus — APPROVED** [review]
Looks good!

**Review by hrizvi1 — APPROVED** [review]
LGTM

File: products/bash/release-ace-integration-server.sh [hunk 1]
```diff
@@ -24,49 +26,63 @@
...
   \?)
     usage
-    exit
     ;;
   esac
 done
 
+echo "INFO: Tracing support currently disabled"
+tracing_enabled=false
+
```
Comment (DanRoseus) [inline products/bash/release-ace-integration-server.sh:76]:
```suggestion
```

**Review by DanRoseus — CHANGES_REQUESTED** [review]
Tracing should not be disabled

**Review by DanRoseus — APPROVED** [review]
Looks good

**Review by Kinshuk1993 — APPROVED** [review]
looks good

**Pull request merged**